index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Material+Icons+Outlined"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Material+Icons+Round"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Material+Icons+Sharp"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Material+Icons+Two+Tone"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!-- <h1>Hello world</h1> -->
+    <div class="main">
+      <header>
+        <nav class="mobile-version">
+          <span
+            class="nav-icon material-icons menu"
+            style="
+              font-size: 55px;
+              color: #444140;
+              font-weight: 600;
+              cursor: pointer;
+            "
+          >
+            menu
+          </span>
+        </nav>
+        <div class="logo">
+          <img src="./Img/Svg/sony-logo-3066.png" alt="" />
+        </div>
+        <nav class="desktop-version" id="show">
+          <ul class="nav-bar">
+            <span
+              class="visible close"
+              style="
+                font-size: 55px;
+                color: #444140;
```

Comment:
On the mobile version, kindly centralize the content of the page.

Your design:
![image](https://user-images.githubusercontent.com/20554583/203770315-a9327dd6-9f9b-4f5f-a988-764cba793b80.png)

Cindy's design:
![image](https://user-images.githubusercontent.com/20554583/203770459-133303d0-79cd-44d6-ab36-70e71103a79d.png)